conda/recipes/pynvjitlink/meta.yaml

```
@@ -0,0 +1,55 @@
+# Copyright (c) 2018-2024, NVIDIA CORPORATION.
+
+{% set data = load_file_data("pyproject.toml") %}
+
+{% if cuda_compiler_version is defined %}
+{% set cuda_major = cuda_compiler_version.split(".")[0]|int %}
+{% set cuda_minor = cuda_compiler_version.split(".")[1]|int %}
+{% else %}
+{% set cuda_major = 0 %}
+{% set cuda_minor = 0 %}
+{% endif %}
+
+package:
+  name: pynvjitlink
+  version: {{ data.get("project", {}).get("version", "") }}
+
+source:
+  path: ../../..
+
+build:
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+  ignore_run_exports_from:
+    - libnvjitlink-dev
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cuda-version {{ cuda_compiler_version }}
+    - cmake >=3.26.4
+    - ninja
+    - sysroot_{{ target_platform }} 2.17
+  host:
+    - libnvjitlink-dev
+    - libnvjitlink-static
+    - cuda-version {{ cuda_compiler_version }}
+    - python
+    - pip
+    - scikit-build-core
+  run:
+    - python
+    - numba >=0.57
+    - {{ pin_compatible('cuda-version', min_pin='x', upper_bound=(cuda_major ~ "." ~ (cuda_minor + 1))) }}
+
+about:
+  home: https://rapids.ai/
+  dev_url: https://github.com/rapidsai/pynvjitlink
+  doc_url: https://github.com/rapidsai/pynvjitlink/blob/main/README.md
+  license: Apache-2.0
```

I don't remember whether we want "Apache 2.0" or "Apache-2.0" (I know there is some context where there is a difference), but I imagine that we want the same in both pip and conda packages so we should make sure that what pyproject.toml has is correct. In case the two actually need to be different for some reason, then ignore this change.
```suggestion
  license: {{ project_data["license"] }}
```